src/domain/hanke/edit/Form.styles.scss

```diff
@@ -1,5 +1,7 @@
+@import '../../common/components/app/layout.scss';
```

Comment:
Move layout.scss to src/assets/styles/ and then you can import it like `@import 'layout.scss'`